src/_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -95,6 +96,17 @@ Parameters:
 are available on [OVH
 documentation](https://docs.ovh.com/fr/logs-data-platform/quick-start/).
 
+#### Logtail
+```bash
+scalingo --app my-app log-drains-add --type logtail \
+  --token 123456789abcdef
+```
+Parameters:
+* `token`: Logtail's Bearer Token
+
+These variable are available on
```

Comment:
I would prefer to refer this page: https://betterstack.com/docs/logs/logging-start/